src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/Travellers/SingleTravellerSelection.tsx

```diff
@@ -33,6 +34,10 @@ export function SingleTravellerSelection({
   const isOnBehalfOfEnabled =
     useOnBehalfOf() && fareProductTypeConfig.configuration.onBehalfOfEnabled;
 
+  const isLoggedIn = useAuthState().authenticationType === 'phone';
```

Comment:
Nit: We mostly implement things like this with `const {authenticationType} = useAuthState();`, that way we don't need to call useAuthState more than once when you need more than one field. Just a note, no blocker for merging the PR :)